samples/amber.cc

```diff
@@ -147,8 +148,11 @@ bool ParseArgs(const std::vector<std::string>& args, Options* opts) {
       opts->parse_only = true;
     } else if (arg == "-d") {
       opts->disable_validation_layer = true;
+    // -s is an old option which is still recognized but does nothing.
     } else if (arg == "-s") {
       opts->show_summary = true;
```

Comment:
Maybe remove `show_summary` (since it seems unused) and make this `quiet = false` (or ignore)?

Reply:
good point, done.